### PR TITLE
fixes RHOARDOC-1171

### PR DESCRIPTION
**Pull request by rhoads-zach**
FYI @inoxx03

### CODE REVIEW

File: docs/topics/proc_show-debug-logging-openshift.adoc [hunk 1]
```diff
@@ -46,10 +46,18 @@ $ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
 ----
 
 . Return to your pod logs and notice there are debug logging messages in the logs.
+ifdef::spring-boot-logging[]
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 i.o.booster.service.GreetingEndpoint     : Message: Hello, Sarah!
 ---- 
+endif::spring-boot-logging[]
```
Comment (Ladicek) [inline docs/topics/proc_show-debug-logging-openshift.adoc]: Should we maybe have examples for the other runtimes? Not in this PR, but for the future.

Also the text above (line 23 specifically) uses `mvn fabric8:deploy` -- is this topic also used in the Node.js guide?

Comment (inoxx03) [reply docs/topics/proc_show-debug-logging-openshift.adoc]: I'm fairly sure that the Node guide contains a different module, specifically `proc_show-debug-logging-openshift-nodejs.adoc`  that covers this topic.

**Comment by inoxx03** [comment]
LGTM, merging...

File: docs/topics/proc_show-debug-logging-openshift.adoc (hunk 1 above)
Comment (Ladicek) [reply docs/topics/proc_show-debug-logging-openshift.adoc]: Ah OK.

**Comment by inoxx03** [comment]
merged: 42df70d8a787bdfadf17d3271305d572fd04b948

**Pull request closed without merging**